assets/js/atomic/blocks/product-elements/shared/config.tsx

```diff
@@ -4,6 +4,7 @@
 import { __ } from '@wordpress/i18n';
 import { Icon, grid } from '@woocommerce/icons';
 import { isExperimentalBuild } from '@woocommerce/block-settings';
+import { BlockConfiguration } from '@wordpress/blocks';
```

Comment:
If you're just importing a type from a module then you can use the `type` keyword.

```suggestion
import type { BlockConfiguration } from '@wordpress/blocks';
```